geode-docs/developing/transactions/design_considerations.html.md.erb

```diff
@@ -117,6 +118,21 @@ immediately after the commit.
 A transaction disables expiration on any region entries affected
 by the transaction.
 
+## <a id="transactions-nontransactions" class="no-quick-link"></a>Mixing Transactions with Nontransactional Operations
```

Comment:
Add hyphen to "Nontransactional" to become Non-transactional.

Reply:
Good catch, thanks.